README.md

```diff
@@ -84,7 +84,7 @@ wipSuite =
         [ only <| describe "Marking this test as `only` means no other tests will be run!"
             [ test "This test will be run" <|
                   \_ -> 1 + 1 |> Expect.equal 2
-            , skip <| "This test will be skipped, even though it's in an only!" <|
+            , skip test <| "This test will be skipped, even though it's in an only!" <|
```

Comment:
Apologies that this isn't properly formatted Elm code (per `elm-format`). I'm just getting started, so I'm not super familiar with the syntax yet 😅